<main class="mdl-layout__content">
<div class="mdl-layout__tab-panel is-active" id="home">
    <div class="mdl-grid">
        <div class="welcome mdl-card mdl-shadow--2dp mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet mdl-cell--12-col-desktop">
            <div class="mdl-card__title mdl-card--expand mdl-color--teal-500">
                <h2 class="mdl-card__title-text mdl-color-text--grey-50">
                    <?php
                        content('welcome');
                    ?>
                </h2>
            </div>
            <div class="mdl-card__supporting-text mdl-color-text--grey-600">
                <h3>Content</h3>
                CajaDB, the most comprehensive knowledgebase in marmosets, provides an intuitive interface to visualize and explore genomic, transcriptomic and alternative splicing data. Our application not only allows the user to navigate the data with neuropsychiatric and neural genes information but also supports biological analyses such as functional (ontology) enrichment analysis and protein-protein-network. Hopefully, these centralized resources will provide numerous benefits to researchers in addressing scientific questions. 
                <br>
                <br>
                More detailed information of this application is provided at Nogueira, VB. Imparato, DO. Souza, SJ. Sousa, MBC. 2017. CajaDB: a knowledgebase of common marmosets (Callithrix jacchus) (In press).
                
                <hr>

                This research was supported by Amazonas State Research Support Foundation (FAPEAM), Federal University of Rio Grande do Norte (UFRN), Brain Institute, Bioinformatics Multidisciplinary Environment (BioME), NPAD/UFRN and Brazilian Council for Research and Technological Development (CNPq Proc. No. 30.6018/2013-6).

                <hr>

                <h3>References</h3>

                Szklarczyk D, Morris JH, Cook H, Kuhn M, Wyder S, Simonovic M, Santos A, Doncheva NT, Roth A, Bork P, Jensen LJ, von Mering C. The STRING database in 2017: quality-controlled protein-protein association networks, made broadly accessible. Nucleic Acids Res. 2017 Jan; 45:D362-68
                <br>
                <br>
                Cortez, D., Marin, R., Toledo-Flores, D., Froidevaux, L., Liechti, A., Waters, P. D., et al. (2014). Origins and functional evolution of Y chromosomes across mammals. Nature 508, 488–93. doi:10.1038/nature13151.
                <br>
                <br>
                Pipes, L., Li, S., Bozinoski, M., Palermo, R., Peng, X., Blood, P., et al. (2013). The non-human primate reference transcriptome resource (NHPRTR) for comparative functional genomics. Nucleic Acids Res. 41, 906–914. doi:10.1093/nar/gks1268.
                <br>    
                <br>     
                Peng, X., Thierry-Mieg, J., Thierry-Mieg, D., Nishida, A., Pipes, L., Bozinoski, M., et al. (2014). Tissue-specific transcriptome sequencing analysis expands the non-human primate reference transcriptome resource (NHPRTR). Nucleic Acids Res. 43, D737–D742. doi:10.1093/nar/gku1110.
                <br>     
                <br>    
                Welter, D., MacArthur, J., Morales, J., Burdett, T., Hall, P., Junkins, H., et al. (2014). The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res. 42, 1001–1006. doi:10.1093/nar/gkt1229.
                <br>   
                <br>  
                Kuleshov, M. V, Jones, M. R., Rouillard, A. D., Fernandez, N. F., Duan, Q., Wang, Z., et al. (2016). Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Res. 44, W90-7. doi:10.1093/nar/gkw377.
                <br> 
                <br>
                Kroll, J. E., Kim, J., Ohno-Machado, L., and de Souza, S. J. (2015). Splicing Express : a software suite for alternative splicing analysis using next-generation sequencing data. PeerJ 3, e1419. doi:10.7717/peerj.1419.

            </div>
        </div>
